.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.9, '3.10']
+        python-version: [3.9, '3.10', '3.11']
```

Comment:
```suggestion
        python-version: ['3.9', '3.10', '3.11']
```
?

Reply:
No real need for it. But yea why not, so it's consistent.